### PR TITLE
Update password-policy-recommendations.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the minimum character length recommendation as per the latest update from https://www.microsoft.com/en-us/research/wp-content/uploads/2016/06/Microsoft_Password_Guidance-1.pdf from Robyn Hicock, rhicock@microsoft.com [Microsoft Identity Protection Team]

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10432

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/misc/password-policy-recommendations.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Good password practices fall into a few broad categories:
 
 The primary goal of a more secure password system is password diversity. You want your password policy to contain lots of different and hard to guess passwords. Here are a few recommendations for keeping your organization as secure as possible.
 
-- Maintain a 14-character minimum length requirement
+- Maintain an 8-character minimum length requirement (and longer is not necessarily better).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/password-policy-recommendations.md]:
```suggestion
- Maintain an 8-character minimum length requirement (longer is not necessarily better).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request closed without merging**